utils/python/CIME/case.py

```diff
@@ -77,6 +78,24 @@ def _get_env(self, short_name):
                   return env_file
           expect(False, "Could not find object for %s in case"%full_name)
 
+    def copy(self, newcasename, newcaseroot, newcimeroot=None, newsrcroot=None):
+        newcase = deepcopy(self)
```

Comment:
This deepcopy is highly suspect. Case is a custom type and python has no idea natively how to deep copy it. See this:
http://stackoverflow.com/questions/1500718/what-is-the-right-way-to-override-the-copy-deepcopy-operations-on-an-object-in-p

Is there any value in making a copy function instead of just creating a new Case object for the clone?